package.json

```diff
@@ -33,8 +34,10 @@
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-promise": "^6.0.0",
     "eslint-plugin-react": "^7.27.1",
+    "express": "^4.17.2",
     "fast-glob": "^3.2.7",
     "flow-bin": "^0.170.0",
+    "flow-enums-runtime": "^0.0.6",
```

Comment:
@jeresig for reference, regarding your comment.

Reply:
Yeah, it just wasn't clear why it moved - since it seems like it would still be needed at runtime?

Reply:
It's the root level package.json. That isn't used by anything when these packages are installed - it's only used during dev.

The individual package.json's are responsible for their dependencies at runtime and since we do rollup and they don't specify this as a peer dep, it gets incorporated into each package that needs it anyway.